pages/linux/llvd.md

```diff
@@ -0,0 +1,20 @@
+# llvd
+
+> Linkedin Learning Video Downloader.
+> More information: <https://github.com/knowbee/llvd>.
+
+- Download a Course:
```

Comment:
```suggestion
- Download an [c]ourse using cookie-based authentication:
```
I think mentioning why we use the cookies flag in the command description would be better.